scripts/extensions/sams/src/components/assets/assetImagePreviewFullScreen.tsx

```diff
@@ -0,0 +1,209 @@
+// External Modules
+import * as React from 'react';
+
+// Types
+import {IAssetItem} from '../../interfaces';
+import {superdeskApi} from '../../apis';
+
+// UI
+import {ButtonGroup, Label} from 'superdesk-ui-framework/react';
+import {
+    Modal,
+    PanelContentBlock,
+    PanelContentBlockInner,
+} from '../../ui';
+import {AssetImageRendition} from './AssetImageRendition';
+import {getPreviewComponent} from './preview';
+import {ToggleBoxNext} from 'superdesk-ui-framework';
+import {VersionUserDateLines} from '../common/versionUserDateLines';
+
+// Utils
+import {
+    showModalConnectedToStore,
+    getHumanReadableFileSize,
+} from '../../utils/ui';
+import {getMimetypeHumanReadable} from '../../utils/assets';
+
+interface IProps {
+    asset?: IAssetItem;
+    closeModal(): void;
+}
+
+export function showImagePreviewModal(asset: IAssetItem) {
+    return showModalConnectedToStore(AssetImagePreviewFullScreen, {asset: asset});
+}
+
+export class AssetImagePreviewFullScreen extends React.Component<IProps> {
+    render() {
+        const {gettext} = superdeskApi.localization;
+        const {config} = superdeskApi.instance;
+        const ContentPreview = getPreviewComponent(this.props.asset!);
+        const mimetype = getMimetypeHumanReadable(this.props.asset?.mimetype);
+
+        return (
+            <Modal
+                id="AssetImagePreviewFullscreenModal"
+                size="fullscreen"
+                closeModal={this.props.closeModal}
+            >
+                {/* Left section, containing image preview. */}
+
+                <section
+                    style={{
```

Comment:
We should avoid using inline `style` attributes, and try and use the UI-Framework as much as possible.

Could you use the classes used from monitoring's fullscreen media preview?
https://github.com/superdesk/superdesk-client-core/blob/develop/scripts/apps/archive/views/media-view.html